### PR TITLE
cradle: add page

### PR DESCRIPTION
**Pull request by pxgamer**
The command line for the [Cradle Framework](https://cradlephp.github.io).

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/cradle.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cradle
+
+> The Cradle PHP framework.
+> See `cradle-install`, `cradle-deploy` and other pages for additional information.
+
+- Connect to a server:
+
+`cradle connect {{connection_name}}`
```
Comment (agnivade) [inline pages/common/cradle.md]: I am guessing this is something like "user:password@hostname:port" ? Can we write something like that to make it clearer ?

Comment (pxgamer) [reply pages/common/cradle.md]: It's just something like `app` or whatever you want it to be called. The details for it are specified in the `config/deploy.php` file.

https://github.com/CradlePHP/Cradle/blob/master/config/deploy.php

File: pages/common/cradle.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# cradle
+
+> The Cradle PHP framework.
+> See `cradle-install`, `cradle-deploy` and other pages for additional information.
+
+- Connect to a server:
+
+`cradle connect {{server_name}}`
+
+- Call general help:
```
Comment (sbrl) [inline pages/common/cradle.md]: `Call` sounds rather odd here. What about `Show` instead? Same for the next example down.

Comment (pxgamer) [reply pages/common/cradle.md]: Yeah, I don't normally add the `help` commands to tldr pages. But I was looking at `git` for the format.

**Pull request merged**